source/ceylon/net/http/server/MatcherRule.ceylon

```diff
@@ -16,6 +17,22 @@ shared abstract class Matcher() {
             => Or(this, other);
 }
 
+"Matcher to leverage Undertow's template mechanism for path templates and path parameters.
+ It should be given to an Endpoint without combining it with other matchers.
+ 
```

Comment:
I cannot examine closely right now (will do this evening), but off of my head the result must be like this:if you OR a PrefixPathMatcher and a TemplateMatcher, the PrefixPathMatcher will match, but the TemplateMatcher not (as the logic to install the undertow template handler just checks if the Ceylon Endpoint has a TemplateMatcher installed, which is not the case if it is an OR matcher.Matej Lazar <notifications@github.com> hat am 5. Februar 2016 um 09:55 geschrieben:In source/ceylon/net/http/server/MatcherRule.ceylon:> @@ -16,6 +17,22 @@ shared abstract class Matcher() {
>              => Or(this, other);
>  }
>  
> +"Matcher to leverage Undertow's template mechanism for path templates and path parameters.
> + It should be given to an Endpoint without combining it with other matchers.
> + 
Does you current solution work with OR and fail only when AND is used? If that is the case, then we can go this way and maybe deprecate AND or leave it as open issue and document it. If neither AND nor OR is working then my vote is to fix it before we merge it in.—Reply to this email directly or view it on GitHub.  